frontend/src/App.js

```diff
@@ -23,7 +23,8 @@ export default function App() {
           <Route path="/create" exact component={Create}/>
           <Route path="/setting" component={Settings}/>
       </Switch>
-    </Router>
+    </Router> */}
+      sdfsdf
```

Comment:
I think you removed the entire frontend here?

Reply:
I updated